docs/DeveloperGuide.md

```diff
@@ -888,17 +893,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
-* 2c. User did not input a valid index after the command.
```

Comment:
why u remove this tho coz invalid command and invalid index got diff error messages

Reply:
also if u add it back in can help change to `valid event index`

Reply:
its still there, u have to expand up

Reply:
> also if u add it back in can help change to `valid event index`

ke yi

Reply:
wait one is user never input an event index, one is user never input a ` valid` event index its like 2 diff points